docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+docker build -t codersrank_extractor:latest .
+docker run -it --mount type=bind,source="$1"/,target="/repo" --mount type=bind,source="$(pwd)"/,target="/app" codersrank_extractor:latest
```

Comment:
Like this, all the repos will have `repo` name. Because of this:
```
def create_repo_entity(self, repo_dir):
        return Repository(os.path.basename(repo_dir.rstrip(os.sep)), self.repo, self.commit_list)
```
So it always gets the last directory name as repo name. It is a problem because if multiple repos are generated with this script all of them will have `repo` name and each upload will overwrite the previous one.

Reply:
Done